tensorpack/tfutils/argscope.py

```diff
@@ -60,3 +62,26 @@ def get_arg_scope():
         return _ArgScopeStack[-1]
     else:
         return defaultdict(dict)
+
+
+def argscope_mapper(func):
+    """Decorator for function to support argscope
+    """
+    @wraps(func)
+    def wrapped_func(*args, **kwargs):
+        actual_args = copy.copy(get_arg_scope()[func.__name__])
+        actual_args.update(kwargs)
+        out_tensor = func(*args, **actual_args)
+        return out_tensor
+    a = wrapped_func
+    # argscope requires this property
+    a.symbolic_function = None
+    return a
+
+
+def enable_argscope_for_lib(lib, decorator=argscope_mapper):
```

Comment:
Why a `decorator` argument?
If the purpose of this function is "enable_argscope" then why would a user want to use a different decorator?

Reply:
"module" may be a better name than "lib"